src/components/PostForm.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  Box,
+  Card,
+  CardBody,
+  CardHeader,
+  Heading,
+  Stack,
+  StackDivider,
+  Text,
+} from "@chakra-ui/react";
+import React from "react";
+import { BlogPostCategory } from "../types/blogPost";
+
+export interface PostFormType {
+  title: string;
+  score: number;
+  categories: BlogPostCategory[];
+  author: string;
+}
+
+export function PostForm({ author, categories, score, title }: PostFormType) {
```

Comment:
I think it is not form, so it should not be named as form. It's more like `PostCard` or something like that